snap/frontend/.eslintrc

```diff
@@ -5,6 +5,9 @@ extends:
 - plugin:jsdoc/recommended-error
 - ../../linters/javascript/default.eslintrc
 rules:
+  eol-last:
```

Comment:
eol-last pluggin seems to be deprecated and should use https://eslint.style/rules/js/eol-last